app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -148,20 +152,17 @@ fun ProfileScreen(
 @Composable
 fun DisplayFriendList(friends: List<User?>) {
   return if (friends.isNotEmpty()) {
-
-    // LazyColumn is scrollable
-    LazyColumn(
-        contentPadding = PaddingValues(vertical = 0.dp),
-        modifier = Modifier.fillMaxWidth().padding(horizontal = 16.dp).testTag("friendList")) {
-          items(friends) { friend ->
-            if (friend != null) {
-              DisplayFriend(friend)
-            }
-          }
+    LazyColumn(modifier = Modifier.fillMaxSize().testTag("friendList")) {
```

Comment:
Note: Good job using `LazyColumn` for displaying friend items in `DisplayFriendList`. Lazy loading is beneficial for handling a potentially large list of friends.